### PR TITLE
 chore: add auth for /graphql route

### PR DESCRIPTION
**Pull request by mprevide**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:
dojot/dojot#1388

### CODE REVIEW

File: auth/initialConf.py [hunk 1]
```diff
@@ -113,16 +113,14 @@ def create_permissions():
         permission_dict_helper('ro_user', "/auth/user/(.*)", "GET"),
         permission_dict_helper('all_pap', "/auth/pap/(.*)", "(.*)"),
         permission_dict_helper('ro_pap', "/auth/pap/(.*)", "GET"),
-        permission_dict_helper('all_x509_ca', "/x509/v1/ca/(.*)", "(.*)"),
-        permission_dict_helper('ro_x509_ca', "/x509/v1/ca/(.*)", "GET"),
-        permission_dict_helper('all_x509_certificates', "/x509/v1/certificates/(.*)", "(.*)"),
-        permission_dict_helper('ro_x509_certificates', "/x509/v1/certificates/(.*)", "GET"),
```
Comment (thiagodpf) [inline auth/initialConf.py:119]: Can these routes really be removed? Won't it impact access to the x509 certificate component?

File: auth/initialConf.py [hunk 2]
```diff
@@ -163,14 +161,12 @@ def add_permissions_group():
                 'all_device',
                 'all_flows',
                 'ro_history',
-                'all_x509_ca',
-                'ro_x509_ca',
-                'all_x509_certificates',
-                'ro_x509_certificates',
```
Comment (thiagodpf) [inline auth/initialConf.py:169]: Should this really be removed?

File: auth/initialConf.py (hunk 1 above)
Comment (mprevide) [reply auth/initialConf.py:119]: No! =O

File: auth/initialConf.py (hunk 2 above)
Comment (mprevide) [reply auth/initialConf.py:169]: No!

**Pull request closed without merging**